### PR TITLE
[IOTDB-775] Add SQL of creating snapshot manually

### PR DESCRIPTION
**Pull request by samperson1997**
Add SQL for users to create snapshot manually.
**Grammar**: `create snapshot for schema`

### CODE REVIEW

**Comment by samperson1997** [comment]
@HTHou Thanks for your review and approval during festival > <

File: server/src/main/java/org/apache/iotdb/db/qp/Planner.java [hunk 1]
```diff
@@ -100,6 +100,7 @@ protected Operator logicalOptimize(Operator operator)
       case CLEAR_CACHE:
       case NULL:
       case SHOW_MERGE_STATUS:
+      case CREATE_SNAPSHOT:
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/Planner.java:103]:
```suggestion
      case CREATE_SCHEMA_SNAPSHOT:
```

File: server/src/main/java/org/apache/iotdb/db/qp/constant/SQLConstant.java [hunk 2]
```diff
@@ -152,6 +152,8 @@ private SQLConstant() {
 
   public static final int TOK_SHOW_MERGE_STATUS = 87;
 
+  public static final int TOK_CREATE_SNAPSHOT = 88;
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/constant/SQLConstant.java:155]:
```suggestion
  public static final int TOK_CREATE_SCHEMA_SNAPSHOT = 88;
```

File: server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java [hunk 3]
```diff
@@ -76,6 +76,6 @@ public String toString() {
     GRANT_WATERMARK_EMBEDDING, REVOKE_WATERMARK_EMBEDDING,
     TTL, DELETE_STORAGE_GROUP, LOAD_CONFIGURATION, SHOW, LOAD_FILES, REMOVE_FILE, MOVE_FILE, LAST, GROUP_BY_FILL,
     ALTER_TIMESERIES, FLUSH, MERGE, FULL_MERGE, CLEAR_CACHE,
-    SHOW_MERGE_STATUS
+    SHOW_MERGE_STATUS, CREATE_SNAPSHOT
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java:79]:
```suggestion
    SHOW_MERGE_STATUS, CREATE_SCHEMA_SNAPSHOT
```

File: server/src/main/java/org/apache/iotdb/db/qp/Planner.java (hunk 1 above)
Comment (samperson1997) [reply server/src/main/java/org/apache/iotdb/db/qp/Planner.java:103]: Fixed

File: server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java (hunk 3 above)
Comment (samperson1997) [reply server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java:79]: Fixed

File: server/src/main/java/org/apache/iotdb/db/qp/constant/SQLConstant.java (hunk 2 above)
Comment (samperson1997) [reply server/src/main/java/org/apache/iotdb/db/qp/constant/SQLConstant.java:155]: Fixed

**Pull request merged**